pymc/tests/test_distributions_moments.py

```diff
@@ -127,6 +128,22 @@ def test_halfnormal_moment(sigma, size, expected):
     assert_moment_is_expected(model, expected)
 
 
+@pytest.mark.parametrize(
+    "nu, sigma, size, expected",
+    [
+        (1, 1, None, 1),
+        (1, 1, 5, np.ones(5)),
+        (1, np.arange(5), None, np.arange(5)),
+        (1, np.arange(5), (2, 5), np.full((2, 5), np.arange(5))),
+        (np.arange(1, 6), 1, 5, np.full(5, 1)),
```

Comment:
This one does not really test the broadcast because size is not None. Maybe something like this?


```suggestion
        (np.arange(1, 6), 1, None, np.full(5, 1)),
```

And you can remove one of the previous 2 tests, we don't need 5 in total.

Reply:
Sure! Thanks for your patience!

Reply:
I thank you :)